python/triton/language/semantic.py

```diff
@@ -1161,20 +1161,20 @@ def validate_descriptor_block(shape, dtype):
 def descriptor_load(desc: tl._experimental_tensor_desciptor_base, offsets, cache_modifier: str, eviction_policy: str,
                     builder: ir.builder) -> tl.tensor:
     assert isinstance(desc, tl._experimental_tensor_descriptor_base)
-    validate_descriptor_block(desc.block_shape, desc.type.element_ty)
+    validate_descriptor_block(desc.block_shape, desc.dtype)
```

Comment:
Small BC-break, `desc.type` is now `tensor_descriptor_type` instead of `block_type`.